.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,7 +31,21 @@ Include specific tasks in the order they need to be done in. Include links to sp
 
 Please follow these checklists to help prevent any unexpected issues from being introduced by the changes in this pull request. If an item does not apply then indicate so by surrounding the line item with `~~` to strikethrough the text. See [basic writing and formatting syntax](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for more information.
 
-### I. Message handlers checklist:
+### I. Web service and data model checklist
```

Comment:
Missed this by accident so sorry about that - there's no web service for the label generator so you can remove references to that in this checklist 

I can make the change in the PR i'm making to your branch

Reply:
👍

Reply:
@divyamadala30 can you make this change? i missed it